en/developer_guide/block_parameter.rst

```diff
@@ -0,0 +1,126 @@
+Block parameter
+===============
+
+Context
+-------
+
+To be properly displayed on the website, some blocks will require environment parameters.
+Those parameters could be of multiple types, for instance :
+
+ - url parameters
+ - server parameters
+ - header parameters
+ - ...
+
+An easy way to give those parameters to the block would have been to give all the request
+parameters when the block is called.
+
+This easy solution is not optimal for a main reason: when loading blocks via `ESI`_, each
+block will be cached for each url variant.
+
+If the block 1 does not require any parameters, in the following exemple :
+
+ * ``/block/1`` will be in one cache entry.
+ * ``/block/1?foo=bar`` will be in another cache entry.
+
+To improve your cache hit ratio, it will be likely to reduce the number of cache
+entry for the same page.
+
+Open Orchestra solution
+-----------------------
+
+Open Orchestra gives you the opportunity to pass only the parameters you want
+to request a block, this way limits the cache entries for a single block.
+
+This solution is decomposed in two part :
+- in the Backoffice, where you add the block
+- in the Frontoffice, where you will require some parameters linked to the block.
+
+Backoffice work
+~~~~~~~~~~~~~~~
+
+In the Backoffice, we provide a way to store the parameter linked to the block in the block reference
```

Comment:
The end of the sentence is not clear to me: what is the meaning of "in the block reference in the areas" ?